components/publish/ProgressBar.tsx

```diff
@@ -48,7 +48,7 @@ const StepBox: React.FC<
   }>
 > = ({ step, current, complete, children }) => {
   return (
-    <div>
+    <div style={{ flex: 1, textAlign: "center" }}>
```

Comment:
can we change this to preferably use Bootstrap, or alternatively a styled component if need be.